libr/util/str.c

```diff
@@ -10,6 +10,76 @@
 #include <ctype.h>
 #include <stdarg.h>
 
+R_API RCharset *r_charset_new(size_t size) {
+	RCharset* rcs = (RCharset*) malloc (sizeof(RCharset));
+	rcs->custom_charset = NULL;
+	rcs->size = size;
```

Comment:
whats the size for?

Reply:
it is the maximal number of different chars contained in the charset.

Reply:
Contrary to remaining what is the offset to the current charset. I am going to remove size. It seems useless finally.